frontend/src/bundles/cv-editor/components/online-editor/online-editor-handler.tsx

```diff
@@ -0,0 +1,42 @@
+import clsx from 'clsx';
+import React, { useCallback, useState } from 'react';
+
+import { type TabsProps as TabsProperties } from '../../types/interface';
+import css from './online-editor-handler.module.scss';
+
+const OnlineEditorTabsHandler: React.FC<TabsProperties> = ({ tabs }) => {
+    const [activeTab, setActiveTab] = useState(0);
+  
+    const handleTabClick = (tabIndex: number) => (event_: React.MouseEvent) => {
+        event_.preventDefault();
+        setActiveTab(tabIndex);
+    };
+
+    const onNextClick = useCallback(() => {
+        setActiveTab(activeTab < tabs.length - 1 ? activeTab + 1 : activeTab);
```

Comment:
Use callback here to prevent getting old value, like: setActiveTab(prevActiveTabIndex => prevActiveTabIndex <.....